Aquality.Selenium.Core/src/Aquality.Selenium.Core/Elements/Interfaces/IElementCacheHandler.cs

```diff
@@ -26,6 +26,6 @@ public interface IElementCacheHandler
         /// <param name="timeout">Timeout used to retrive the element when <see cref="IsRefreshNeeded(ElementState?)"/> is true.</param>
         /// <param name="customState">Custom element's existance state used for search.</param>
         /// <returns>Cached element.</returns>
-        RemoteWebElement GetElement(TimeSpan? timeout = null, ElementState? customState = null);
+        WebElement GetElement(TimeSpan? timeout = null, ElementState? customState = null);
     }
 }
```

Comment:
please add a new line at the end of file

Reply:
added